docs/UserGuide.md

```diff
@@ -227,26 +248,34 @@ through as you are not allowed to add with max quantity and metric. You should r
     Figure 6: Error for invalid input
     <br>
  
+    Your command should not pass through as you are not allowed to add an existing item with max quantity and metric.
+    
    *Note: If you want to edit the maximum quantity or metric of an existing item, refer to [edit](#edit) feature*
 
-5. Repeat step 1 and 2 above but with the command: `add-i n/DUCK q/10 s/NTUC t/perishable`. Notice that you 
-are adding to an existing `item` DUCK which was just added. Furthermore, a new tag of perishable should also appear.
-You can refer to the diagram below for the expected result:
+5. Repeat step 1 and 2 above but with the command: `add-i n/DUCK q/10 s/NTUC t/perishable`. 
+   You can refer to Figure 7 for the expected result:
     <br>
     
     ![add-i_step3](images/add-i_eg1.3.PNG)
     <br>
     
     Figure 7: Successfully added quantity with new tag
     
+    Notice that you 
+    are adding to an existing `item` DUCK which was just added. A new tag of perishable should also appear.
+    
+    
 
 <div markdown="span" class="alert alert-primary">:bulb:**Tip:** <br>
 An item can have any number of tags (including 0) and adding exactly the same tag to an existing item will not result in duplicated tags
 </div>
 
 ##### 3.1.2b `add-d`
 
-Format: `add-d n/NAME p/PHONE a/ADDRESS o/ORDER [by/]`
+Format: `add-d n/NAME p/PHONE a/ADDRESS o/ORDER [by/TIME]`
+
+Pointers to take note when adding a delivery:
+* You can add a delivery even if the same `Name`, `Phone` and `Address` already exists in the delivery book.
```

Comment:
Do you think we should mention in the pointers that time needs to be in minutes?

Reply:
Good point, added ur suggestion pls check again